src/modules/Dashboard/features/Managers/Popups/AddManagerPopup/AddManagerPopup.tsx

```diff
@@ -0,0 +1,227 @@
+
+import { USER_ALREADY_INVITED, defaultValues } from './AddManagerPopup.const';
+import { AddManagerPopupSchema } from './AddManagerPopup.schema';
+import { AddManagerPopupProps, AddManagerFormValues, Fields } from './AddManagerPopup.types';
+import { AddManagerForm } from './AddManagerForm';
+
+export const AddManagerPopup = ({
+  appletId,
+  onClose,
+  popupVisible,
+  'data-testid': dataTestid,
+}: AddManagerPopupProps) => {
+  const { t, i18n } = useTranslation('app');
+  const rolesData = workspaces.useRolesData();
+  const appletRoles = appletId ? rolesData?.data?.[appletId] : undefined;
+
+  const [workspaceInfo, setWorkspaceInfo] = useState<WorkspaceInfo | null>(null);
+  // Workspace Name field is only available if the workspace has no managers yet
+  const isWorkspaceNameVisible = !!workspaceInfo && !workspaceInfo.hasManagers;
+
+  const { ownerId } = workspaces.useData() || {};
+  const respondentsData = users.useAllRespondentsData();
+  const participants = (respondentsData?.result ?? []).map(({ details }) => {
+    const { respondentSecretId, respondentNickname, subjectId } = details[0];
+
+    return {
+      subjectId,
+      secretId: respondentSecretId,
+      nickname: respondentNickname,
+      // TODO: Populate tag
+      // https://mindlogger.atlassian.net/browse/M2-5861
+      tag: null,
+    };
+  });
+
+  const defaults = {
+    ...defaultValues,
+    language: i18n.language as Languages,
+    workspaceName: workspaceInfo?.name,
+  };
+
+  const {
+    control,
+    formState: { isValid, isDirty },
+    handleSubmit,
+    register,
+    reset,
+    setError,
+    setValue,
+    unregister,
+  } = useForm<AddManagerFormValues>({
+    resolver: yupResolver(AddManagerPopupSchema(isWorkspaceNameVisible)),
+    defaultValues: defaults,
+    mode: 'all',
+    delayError: VALIDATION_DEBOUNCE_VALUE,
+  });
+
+  const role = useWatch({ control, name: Fields.role });
+
+  const dispatch = useAppDispatch();
+  const [hasCommonError, setHasCommonError] = useState(false);
+
+  const handleClose = (shouldRefetch = false) => {
+    resetForm();
+    setHasCommonError(false);
+    onClose?.(shouldRefetch);
+  };
+
+  const resetForm = () => reset(defaults);
+
+  const {
+    error,
+    execute: createInvitation,
+    isLoading,
+  } = useAsync(postAppletInvitationApi, async (result) => {
+    ownerId && executeGetWorkspaceInfoApi({ ownerId });
+    const { firstName, lastName, title } = result.data?.result ?? {};
+    dispatch(
+      banners.actions.addBanner({
+        key: 'AddParticipantSuccessBanner',
+        bannerProps: {
+          id: `${firstName} ${lastName}${title ? `, ${title}` : ''}`,
+        },
+      }),
+    );
+    Mixpanel.track('Invitation sent successfully');
+    handleClose(true);
+  });
+  const { execute: executeGetWorkspaceInfoApi } = useAsync(getWorkspaceInfoApi, (res) => {
+    setWorkspaceInfo(res?.data?.result || null);
+  });
+
+  const handleSubmitForm = (values: AddManagerFormValues) => {
```

Comment:
I think the columns should be correct now, since I removed the checkbox column?

As for the title, yes, reviewers weren't having their title saved due to the BE bug mentioned in the description, however I just [merged a fix](https://github.com/ChildMindInstitute/mindlogger-backend-refactor/pull/1273) for it, and it should be getting accepted now!